react/index.tsx

```diff
@@ -0,0 +1,88 @@
+import React, { Fragment, FC, useRef, useEffect } from 'react'
+import { generateBlockClass, BlockClass } from '@vtex/css-handles'
+//@ts-ignore
+import styles from './styles.css'
+import { path } from 'ramda'
+
+enum POSITIONS {
+  BOTTOM = 'BOTTOM',
+}
+
+interface Props {
+  position?: POSITIONS
+}
+
+interface StorefrontComponent extends FC<Props & BlockClass> {
+  schema?: any
+}
+
+const StickyLayoutComponent: StorefrontComponent = ({
+  children,
+  position,
+  blockClass,
+}) => {
+  const container = useRef<HTMLDivElement>(null)
+  const startTop = useRef<number | null>(null)
+  const hasTouched = useRef<boolean>(false)
+
+  if (position !== POSITIONS.BOTTOM) {
+    // Only 'BOTTOM' position supported for now!
+    return null
+  }
+
+  const handleScroll = () => {
+    if (!container.current) {
+      return
+    }
+    if (startTop.current == null) {
+      startTop.current = container.current.offsetTop
+    }
+    const newTop =
+      window.innerHeight -
+      startTop.current -
+      container.current.clientHeight +
+      window.pageYOffset
+
+    if (newTop < 0) {
+      hasTouched.current = true
+      container.current.style.transform = `translateY(${newTop}px)`
```

Comment:
it's a hack, but `translate3d(x,y,z)` has generally better performance (bc it forces the element to be in a separate composition layer)